src/SnowballFight.java

```diff
@@ -36,6 +40,11 @@ private void setScene(String tag) {
         currentScene.initialize();
     }
 
+    private void initBackgroundMusic() {
+        backgroundMusic = new AudioClip(BACKGROUND_MUSIC_PATH);
+        backgroundMusic.loop();
+    }
+
     @Override
     public void draw() {
         drawBackground(Colors.WHITE);
```

Comment:
Colors.WHITE is already definded as SCOREBOARD_FRONT_COLOR why don't use the variable?